manifest.json

```diff
@@ -86,23 +86,23 @@
               "dockerfile": "2.0/stretch/runtime",
               "os": "linux",
               "tags": {
-                "2.0.0-stretch": {},
+                "2.0.3-stretch": {},
                 "2.0-stretch": {}
               }
             },
             {
               "dockerfile": "2.0/nanoserver-sac2016/runtime",
               "os": "windows",
               "tags": {
-                "2.0.0-nanoserver-sac2016": {},
+                "2.0.3-nanoserver-sac2016": {},
                 "2.0-nanoserver-sac2016": {},
-                "2.0.0-nanoserver": {
+                "2.0.3-nanoserver": {
                   "isUndocumented": true
                 },
                 "2.0-nanoserver": {
                   "isUndocumented": true
                 },
-                "2.0.0-nanoserver-$(nanoServerVersion)": {
+                "2.0.3-nanoserver-$(nanoServerVersion)": {
```

Comment:
I think these nanoServerVersion suffixed tags should be dropped now.  Because of the reliance on the stable tags such as `2.0.3-sdk-nanoserver-sac2016`, you no longer know with certainty what patch version these images are based on.